### DISCUSSION TITLE
Is there a way to connect directly to the wifi

### DISCUSSION THREAD

**Discussion by aphlox** [outdoorbits/little-backup-box#151] [Q&A]
Is there a way that we can use it as an hotspot so we can connect to pi directly when there is no wifi available?

I have tried install and configure and hotspot but didn't work. I think there is a script that overtakes the network configuration.

**Comment by outdoorbits** [comment]
It works as a hotspot. Just select to install comitup while installation process. Connect to the wlan little-backup-box-nn and browse http://10.41.0.1
Does it work?

**Reply by aphlox** [reply]
Yes, indeed it worked. I wasn't seeing it because I configured it to local wlan while it was connected over ethernet. Once it's disconnected little-backup-box-nn reappeared again. Thanks for the good work.

**Comment by outdoorbits** [comment]
If you browse to https://IP or http://IP:8000 it does not disappear. http://IP just disappears while the comitup web GUI is active.
